core/adminserver/admin_server.go

```diff
@@ -83,8 +83,14 @@ var (
 type LogsAdmin interface {
 	// AddLogs creates and adds new logs for writing to a directory.
 	AddLogs(ctx context.Context, directoryID string, logIDs ...int64) error
+	// SetWritable enables or disables new writes from going to logID.
+	SetWritable(ctx context.Context, directoryID string, logID int64, enabled bool) error
+	// ListLogs returns a list of logs, optionally filtered by the writable bit.
+	ListLogs(ctx context.Context, directoryID string, writable bool) ([]int64, error)
 }
 
+var _ pb.KeyTransparencyAdminServer = &Server{}
```

Comment:
what is this line doing?

Reply:
This line ensures that `&Server{}` implements the `pb.KeyTransparencyAdminServer` interface.

Reply:
I'll add a comment to that effect.

Reply:
ah that's clever ok